src/request.js

```diff
@@ -288,6 +288,17 @@ export function requestUse() {
 	});
 }
 
+export function mockReturn(key, value) {
+	startMockingRequests();
+	requestUse().subscribe(({req, reply}) => {
+		reply({
+			header: {...req.headers, statusCode: 200},
+			body: {[key]: value}
```

Comment:
maybe add an option to provide the entire body, instead of just one key and value, in case that's what you need to mock